inspector/src/components/sceneExplorer/entities/cameraTreeItemComponent.tsx

```diff
@@ -38,10 +38,6 @@ export class CameraTreeItemComponent extends React.Component<ICameraTreeItemComp
         const camera = this.props.camera;
         const scene = camera.getScene();
         this._onActiveCameraObserver = scene.onActiveCameraChanged.add(() => {
-            if (this.state.isActive) {
-                camera.detachControl(scene.getEngine().getRenderingCanvas()!);

```

Comment:
@deltakosh I removed this as this is what was causing the detach of camera drag. There seems to be no re attach code and am unsure why this is being detached in the first place. This codepath seems to get hit on every frame when scene.activeCameras has > 1 camera